torch/fx/graph_module.py

```diff
@@ -515,6 +533,15 @@ def wrapped_call(self, *args, **kwargs):
 
         return python_code
 
+    # Passing Tracer as argument allows subclasses extending fx.GraphModule
+    # define their own Tracer (extending fx.Tracer).
+    def __reduce_deploy__(self, importer: PackageImporter, clsTracer: Any = None):
```

Comment:
What's the motivation behind making `clsTracer` an `Any` as opposed to `Tracer`?

Reply:
We need to pass a class object, not the actual Tracer object . class object argument allows subclasses extending fx.GraphModule define their own Tracer (used in _deserialize_graph_module).

For moving to Type[Tracer] instead of Any, we would need to import Tracer module globally here, but it would introduce a circular dependency. Thats why in _deserialize_graph_module it is imported before using.

The best we can do here IMO is to move from Any to Type.